### PR TITLE
Frontend Adjustments for ABDM Plug Conversion

### PR DESCRIPTION
**Pull request by khavinshankar**
This pull request introduces necessary frontend changes to accommodate the backend conversion of ABDM into a plug. The specific changes include:

- **Fixes to Breaking Changes:** Addressed and corrected several breaking changes that were introduced in the backend. These fixes ensure that the application maintains its functional integrity and provides a stable user experience without disruptions.

- **Enhanced Type Support:** Improved the type definitions across the frontend codebase. This enhancement involves refining the use of TypeScript to ensure more robust and error-resistant code. It improves the predictability of code behavior through better compile-time checks and editor integrations.

- **Removal of Unwanted Console Logs:** Cleaned up various console logs throughout the application that were no longer needed. This removal not only declutters the console output but also potentially improves performance by reducing unnecessary processing and output during runtime.

Depends on: https://github.com/coronasafe/care/pull/2161

### CODE REVIEW

**Comment by nihal467** [comment]
@khavinshankar testing is kept on hold for this PR, due to the ABDM sandbox mobile app crashing. Once the service is back online, ping me, we will get it tested ASAP

**Comment by nihal467** [comment]
@khavinshankar 

image

the consent patient Abha ID is not verifying, and the push notifications are not being triggered in the applications, of chrome

File: netlify.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ NODE_OPTIONS = "--max_old_space_size=4096"
 
 [[redirects]]
 from = "/api/*"
-to = "https://careapi.ohc.network/api/:splat"
```
Comment (shivankacker) [inline netlify.toml]: i think this change was unintentional

**Review by shivankacker — CHANGES_REQUESTED** [review]

**Comment by nihal467** [comment]
@khavinshankar 
image

the consent api is failing, can you check it

**Comment by nihal467** [comment]
@khavinshankar can you fix the cypress failure

**Comment by nihal467** [comment]
LGTM

**Review by shivankacker — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by khavinshankar** [comment]
Closing this PR in favor of #8340

**Pull request merged**